src/components/charts/scatter-plot/scatter-plot-component.jsx

```diff
@@ -98,9 +98,14 @@ function ScatterPlot({
     if (!_data || !selectedKey) return null;
     const highValue = d3.max(_data, (d) => d.xAxisValues[selectedKey]);
     const lowValue = d3.min(_data, (d) => d.xAxisValues[selectedKey]);
+
+    const margin = (highValue - lowValue) / 20;
     // eslint-disable-next-line no-underscore-dangle
     const _formatFunction = countryChallengesChartFormats[selectedKey];
-    return [_formatFunction(lowValue), _formatFunction(highValue)];
+    return [
+      _formatFunction(lowValue - margin),
+      _formatFunction(highValue + margin),
```

Comment:
I think this is probably causing values to go over 100%
<img width="670" alt="image" src="https://user-images.githubusercontent.com/9701591/218515964-e9df9ec8-5187-46dd-89d0-e6ed9f3c7078.png">

http://localhost:3000/nrc/STP/challenges?ui=%7B%22countryChallengesSelectedFilter%22%3A%22filter_steward%22%2C%22countryChallengesSelectedKey%22%3A%22protection_needed_ter%22%7D

Reply:
New solution founded:
Should we add it to yAxis? Is a little weird to have the 0 value up to the Axis.

<img width="460" alt="Screenshot 2023-02-13 at 20 28 30" src="https://user-images.githubusercontent.com/51995866/218555137-82b5eadc-190e-4692-ae5e-f8d443dff949.png">